### ISSUE TITLE
Deserializing views does not work if member delimiter is not single space

### ISSUE THREAD

**Issue by jcklie** [dkpro/dkpro-cassis#115]
**Describe the bug**

Deserializing views does not work if member delimiter is not single space

**Error**

```
/Users/Dima/Git/Sandbox/PyCtakes/>cas_mult_views.py
Traceback (most recent call last):
  File "./cas_mult_views.py", line 11, in <module>
    cas = load_cas_from_xmi(xmi_file, typesystem=type_system)
  File "/usr/local/lib/python3.6/site-packages/cassis/xmi.py", line 40, in load_cas_from_xmi
    return deserializer.deserialize(source, typesystem=typesystem)
  File "/usr/local/lib/python3.6/site-packages/cassis/xmi.py", line 77, in deserialize
    proto_view = self._parse_view(elem)
  File "/usr/local/lib/python3.6/site-packages/cassis/xmi.py", line 215, in _parse_view
    members = [int(e) for e in attributes.get("members", "").strip().split(" ")]
  File "/usr/local/lib/python3.6/site-packages/cassis/xmi.py", line 215, in <listcomp>
    members = [int(e) for e in attributes.get("members", "").strip().split(" ")]
ValueError: invalid literal for int() with base 10: ''
```

**Comment by GregSilverman** [comment]
@jcklie , we just encountered this same error. Is there an ETA for a fix?

**Comment by GregSilverman** [comment]
Hi, we actually have a case where we're getting an error of `invalid literal for int() with base 10: 'DECIMAL'`, too, along with the empty string `''`.

**Comment by GregSilverman** [comment]
Please let me know if you need anything from us. Unfortunately, our XMI are fully identified.

**Comment by GregSilverman** [comment]
I assume this is related to #93.

**Comment by jcklie** [comment]
Did you test whether the error does not occur in master?

**Comment by GregSilverman** [comment]
Will do and let you know the results.

**Comment by GregSilverman** [comment]
@jcklie We just tested it and everything worked as expected! Thanks a ton.

**Comment by jcklie** [comment]
I released 0.2.8, please close this issue if this solves your issue.

**Comment by GregSilverman** [comment]
I would close this if I could, but there isn't the option.
<img width="872" alt="Screen Shot 2020-02-18 at 9 10 54 AM" src="https://user-images.githubusercontent.com/1007034/74748734-9fba9580-522e-11ea-89d1-53d078a9df30.png">
